app/assets/stylesheets/application.scss

```diff
@@ -7,6 +7,14 @@ body {
   height: 100%;
 }
 
+.light-theme {
+  background: white;
+
+  .light-text-white {
```

Comment:
Man ir aidozmas, kā šeit tu vareji droši visus h1-h6 un button tekstus kā melnus radit. Un ši klase tev nav nepiciešama

Reply:
ja es liksu vienkarsi tagus, tad daudzi elementi kurus man vajag atstat baltus ari paliks melni un bus loti slikti saredzami uz background, tapec es to klasi izveidoju uz pievienoju tikai tiem elementiem kuriem ir paredzets nomainit teksta krasu.